Client/src/components/common/Label/index.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 
 const Label = props => {
-  const { input: Input, id, labelText, labelClassName, name, placeholder, onChange, inputClassName} = props;
+  const { input: Input, id, labelText, labelClassName, name, placeholder, onChange, inputClassName, Error, type } = props;
   return (
     <label htmlFor={id} className={labelClassName} >
       {labelText}
-      <Input name={name} placeholder={placeholder} onChange={onChange} inputClassName={inputClassName} />
+      <Input name={name} id={id} type={type} placeholder={placeholder} onChange={onChange} inputClassName={inputClassName} />
```

Comment:
@Fatmasiam Hi, I know this wasn't really part of your PR but I noticed that `inputClassName` should just be applied to this input tag as its classname attribute.  In other words, like this: ` ... onChange={onChange} classname={inputClassname} ...`.  It's not an urgent change to do for this PR, but something to keep in mind for the future.